### DISCUSSION TITLE
Provide multiple table-linked ID fields?

### DISCUSSION THREAD

**Discussion by dan-burt** [Budibase/budibase#8703] [Help]
I have 1 page with a table of data rows. I would like to click any particular row to open a new page (even modal, not that deep into the design yet) linked to which row had been clicked.

However, this data query needs 2 input parameters. The default options is to provide 1. It is named as this:

`ID column for linking (appended to URL)`

If not possible with standard usage, is there any workaround?

**Comment by melohagan** [comment]
Hey @dan-burt 

Instead of linking the table rows, you can add a button with a **Navigate To** action (or a link added to the table component) which would allow you to use more than one variable.
There's an example here: https://github.com/Budibase/budibase/discussions/5299#discussioncomment-2530261
